model/sbgn/BachmannPD_tidynew_color.nwt

```diff
@@ -0,0 +1,1090 @@
+    </arc>
+    <arc id="nwtE_1c522265-f885-44d3-b304-3aa51421c1c2" class="production" source="nwtN_923069d5-14d3-416d-9a83-b83e559835a9" target="nwtN_90d9594e-5d75-4342-b47d-de28cf8d5b3c">
+      <start x="-407.4011616328535" y="-2683.325843099272"/>
+      <end x="-624.519826302172" y="-2650.223754917895"/>
+    </arc>
+    <arc id="nwtE_fd29f465-6a0c-4e9f-a66b-074aa3cdfd8d" class="inhibition" source="nwtN_4f2e2e03-b3b3-4195-9408-3ad001cfed16" target="nwtN_a64979d1-ab09-422d-82fb-a840e8cc45bd">
+      <start x="-38.96517067187918" y="-2092.6932338101096"/>
+      <end x="-175.82948052791915" y="-2092.6499119483105"/>
+    </arc>
+    <arc id="nwtE_8f6a47dc-6e7c-492e-99f9-27c0559d7e60" class="production" source="nwtN_a64979d1-ab09-422d-82fb-a840e8cc45bd" target="5d75f49f-297c-3ca0-3f1e-20780cd2babd">
+      <start x="-203.07948047782304" y="-2092.102489587714"/>
+      <end x="-835.3997130522293" y="-2065.944841881494"/>
+    </arc>
+    <arc id="nwtE_5cf441bb-7bf3-4437-9ad0-8019bd91ee3a" class="consumption" source="4318b000-7413-a7d1-def5-095077bc4f8c" target="nwtN_a64979d1-ab09-422d-82fb-a840e8cc45bd">
+      <start x="-837.1148081368851" y="-2406.258881878152"/>
+      <end x="-203.07948047782304" y="-2099.005806736826"/>
+    </arc>
+    <arc id="nwtE_00395f72-25ef-4f62-b129-2133b1ad4e65" class="production" source="nwtN_38a0bb2c-d5a5-4522-8015-0f705d537167" target="e08d1628-0d6a-e2e7-8532-ece8f502e322">
+      <start x="-203.07948047782304" y="-2013.1519964501845"/>
+      <end x="-835.4536680964665" y="-1891.1026419268233"/>
+    </arc>
+    <arc id="nwtE_5583c9db-ae7c-416e-a27d-364168bf1281" class="consumption" source="a0eb5581-288d-95d4-25af-081d9b9eb23b" target="nwtN_38a0bb2c-d5a5-4522-8015-0f705d537167">
+      <start x="-838.5220425910037" y="-2227.310802631498"/>
+      <end x="-203.07948047782304" y="-2019.9677933169794"/>
+    </arc>
+    <arc id="nwtE_42b49322-b896-462a-aa7a-a5e2b2a0f144" class="inhibition" source="nwtN_cc5fed2c-e3b6-4c40-9967-204d4b21c6fb" target="nwtN_38a0bb2c-d5a5-4522-8015-0f705d537167">
+      <start x="-38.942046221108654" y="-2007.460006054186"/>
+      <end x="-175.83096049276847" y="-2014.9158844412175"/>
+    </arc>
+    <arc id="nwtE_cb73aec2-24c0-4f18-a566-d537e8e42266" class="consumption" source="5d75f49f-297c-3ca0-3f1e-20780cd2babd" target="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c">
+      <start x="-838.5220425910039" y="-2052.3364969389927"/>
+      <end x="-203.07948047782304" y="-1941.0290032656092"/>
+    </arc>
+    <arc id="nwtE_10fbe951-8acf-451a-a1dd-e29773c9a579" class="production" source="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c" target="e08d1628-0d6a-e2e7-8532-ece8f502e322">
+      <start x="-203.07948047782304" y="-1937.6177608361597"/>
+      <end x="-835.4082023335773" y="-1884.0349365442985"/>
+    </arc>
+    <arc id="nwtE_3d4edd18-e61f-4506-ac4c-3f97b0b2a5b6" class="inhibition" source="nwtN_5096a7f8-9215-47f2-8250-00dd6a3813e5" target="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c">
```

Comment:
Maybe also the arcs from SOCS3 and CIS could be colored

Reply:
Please review updated map c4da95a